packages/react-ui-components/src/Headline/headline.tsx

```diff
@@ -7,10 +7,10 @@ type HeadlineType = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
 
 interface HeadlineTheme {
     readonly 'heading': string;
-    readonly 'heading--h1': string;
+    readonly 'heading--h1': string; // TODO ?
```

Comment:
I'm not sure what this does.
Does this have a purpose?

Reply:
Only @Inkdpixels would know for sure, but I think it was meant to be used to style different variants of a headling (see `style` prop)

Reply:
Ok, but there's only the `h1` one. I'm not even sure if it ever worked like this 😅.

I could either remove this or add theming for the other 5 HeadlineTypes and toggle them accordingly.

Reply:
I'd say that since for now we don't use it anywhere it'd be safe to remove it. If we'd ever need it, we could always re-introduce it properly.